### DISCUSSION TITLE
tight screenshots

### DISCUSSION THREAD

**Discussion by nschloe** [pyvista/pyvista#4913] [Q&A]
When taking `p.screenshot()`s with pyvista, one usually has wide margins, e.g.,

![srgb-gamut-slice-cielab](https://user-images.githubusercontent.com/181628/130812408-7930974e-c07f-4ff7-8a51-2a359324d7be.png)

How to remove those? ([matplotlib has `bbox_inches="tight"` for this purpose.](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.savefig.html))

**Comment by palatyle** [comment]
Also have this question. I've found you can get around this somewhat by fiddling with the `window_size` arg in `pv.Plotter()` and changing the zoom level of the camera using `p.camera.zoom(val)`.

**Comment by MaNyh** [comment]
I have the same question. I'd like to save some screenshots in a workflow. It would be cool not to plot areas that are not of interest.

Is there an equivalent way to matplotlib.pyplot.tight_layout in plotting?

**Comment by dodu94** [comment]
I think Plotter.camera.zoom('tight') does more or less what we are looking for.

Legend will be superimposed to the figure though in this way.
